packages/html-plugin/index.js

```diff
@@ -70,12 +70,20 @@ const defaultTemplate = ({
 <html>
   <head>
     <meta charset='utf-8'>
-    <meta name='viewport' content='width=device-width,initial-scale=1'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0'/>
     <style>*{box-sizing:border-box}body{margin:0;font-family:system-ui,sans-serif}</style>
     <meta name='generator' content='mdx-deck'>
+    <link rel="manifest" href="/manifest.json">
+    <script async="true" src="https://cdn.jsdelivr.net/npm/pwacompat@2.0.9/pwacompat.min.js" integrity="sha384-VcI6S+HIsE80FVM1jgbd6WDFhzKYA0PecD/LcIyMQpT4fMJdijBh0I7Iblaacawc" crossorigin="anonymous"></script>
```

Comment:
I'm sure this is fairly harmless, but I don't want to load third-party scripts like this. What is this for?

Reply:
`pwacompact` handles the browsers nuisance for PWA. It can be loaded as a script or can be part of the bundle too using `import`